Add an initial job blog post

ericholscher opened the pull request.

ericholscher committed:
Add an initial job blog post

ericholscher committed:
Fix up feedback request

ericholscher committed:
Small transition improvement

ericholscher committed:
Kill more useful words

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+Write the Docs gets you a job
```
I reckon we need a punchier poppier title :-p

Find a job you love through Write the Docs
===================================

I'm sure @kelly-o has ideas about this

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+Write the Docs gets you a job
+=============================
+
+Write the Docs is a community for everyone who cares about documentation.
```
s/is/the/ ? :-p

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+=============================
+
+Write the Docs is a community for everyone who cares about documentation.
+Over time,
```
How about

> From the beginning we have been connecting documentarians with companies that need people to solve documentation related problems.

plaindocs replied:
Or maybe "from the beginning" is too vague. "Ever since we started WTD"...

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+we have brought together people from myriad companies and job titles to share in their passion for communication and helping users.
+Our in person and online spaces provide a wonderful place to learn best practices around documentation.
+Because of this,
+we have attracted a large number of people at the forefront in thinking around documentation.
```
"people at the forefront in thinking" ? hmmm :/
thought leaders? hmmm :/

> Because of this, our community is composed of many people at the forefront of the documentation industry? 

I don't like that :arrow_up:  either.

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+Our in person and online spaces provide a wonderful place to learn best practices around documentation.
+Because of this,
+we have attracted a large number of people at the forefront in thinking around documentation.
+These best practices continue to push the industry forward,
```
> These best practices, shared at conferences and online,  continue to push the industry forward.

?

plaindocs replied:
And then maybe straight into 

> We also know that each person in our community wants to move their career forward. They want to work with forward thinking companies. They want coworkers who *get it*, coworkers who also care about their customers

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+People in our community want to work with forward thinking companies.
+They want coworkers who *get it*,
+who care about their customers as much as they do.
+We have been seeing this happen organically in our *#jobs* channel in Slack for years now,
```
hmmm, this doesn't really follow. How about

> This connection has been happening organically in our *#jobs* channel in Slack for years now,

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+
+The Job Board
+-------------
+
```
Do we wanna caps Job Board throughout? Or not?

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+The Job Board
+-------------
+
+We are launching a new Job Board: http://jobs.writethedocs.org.
```
We are launching a new Job Board http://jobs.writethedocs.org that has two simple functions:

* Employers can post open positions
* Candidates can search for jobs and set up alerts

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+* Allows employers to post jobs
+* Allows employees to get alerted when jobs are posted
+
+In order to get jobs out to our full community,
```
Of course we also hightlight jobs in our Newsletter and in the *#jobs* channel in Slack.

OT: Do we want to add a Tweet option here?

kelly-o replied:
@ericholscher Could we adapt the RSS fanciness you did with Slack to just, like, auto-tweet any new featured job?

ericholscher replied:
Probably. We might want a bit more editorial control over our main twitter accnt though, but I could be convinced.

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+In order to get jobs out to our full community,
+we'll also be highlighting some of the jobs in our Newsletter and in the *#jobs* channel in Slack.
+
+Job postings are currently priced:
```
insert "as follows"? one of the few occasions where it makes sense?

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+* $50 for a basic job posting (posted to the Web, Slack, and Email)
+* $150 for a promoted job posting (also included in our Newsletter)
+
+We're excited to have this ability to conenct our community with jobs.
```
We're excited to have more and better ways to connect people, and we're confident that this will move us towards our goal to improve documentation in the software industry.

plaindocs commented on docs/blog/write-the-docs-gets-you-a-job.rst:
```diff
@@ -0,0 +1,80 @@
+We truly believe that the folks in our community are pushign the industry forward,
+and helping to facilitate this activity will make only make documentaiton in the software industry get better even quicker!
+
+We're also formalizing the hiring processes at our conference this year,
```
This feels throw away here. Do we want to mention two ininitiatives in the opening para, as introduction to the two sections?

plaindocs reviewed (changes requested):
This is a rocking start.

Hope I haven't gone overboard on the suggestions. :-)

kelly-o committed:
round 1 – incorporating edits

kelly-o reviewed (approved):
incorporated all these – next up is a few edits of my own ;)

kelly-o committed:
round 2 – KOB edits

I went to town – I hope that’s okay

@ericholscher I’d love to have your eyes on it before we push go. Would
you feel okay about doing a co-author byline?

ericholscher commented:
@kelly-o shared by-line sounds good, if our blog can support it. Otherwise happy to let you have it :)

ericholscher committed:
Fix up a few small things

kelly-o committed:
added bloggity things

kelly-o commented:
posted!
